### DISCUSSION TITLE
Border radius bug

### DISCUSSION THREAD

**Discussion by Jan-DT** [tauri-apps/tauri#11978] [Q&A]
Whenever I try adding a border-radius on a button on Ubuntu WSL, I keep running into the following issue:
![Screenshot 2024-12-15 142905](https://github.com/user-attachments/assets/85ccc4a0-f0b7-4f16-8a88-9475cda073fe)
Is this a known bug, or should I report this (either to Tauri or I guess WebkitGtk)?

**Comment by FabianLars** [comment]
I don't think i've seen this before tbh so if it's a bug it seems to be a rare one. fwiw, i used border-radius in my old apps just fine.

**Comment by Jan-DT** [comment]
Maybe this just happens when using WSL? Could be something with the way WSL runs WebkitGtk.
Building for Windows and running the .exe does not have this issue (kinda to be expected).

Would this be an issue to report to Tauri?

**Reply by FabianLars** [reply]
I was thinking only about Linux in my message. And no, an issue report in Tauri itself would be fairly useless since we don't work on webkitgtk.
